lib/internal/Magento/Framework/File/Uploader.php

```diff
@@ -140,14 +140,14 @@ class Uploader
      * @deprecated
      * @see \Magento\Framework\Image\Adapter\UploadConfigInterface::getMaxWidth()
      */
-    const MAX_IMAGE_WIDTH = 4096;
+    const MAX_IMAGE_WIDTH = 'dev/image/max_width';
```

Comment:
You should not replace a constant value in this way. A 3rd party extension could have used it (even if it is deprecated now).

Reply:
hi @phoenix128  , thanks for your review , i am going to fix that